packages/react/src/Label/Label.tsx

```diff
@@ -0,0 +1,104 @@
+import clsx from 'clsx'
+import React, {forwardRef, type Ref} from 'react'
+import {Text} from '../Text'
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/label/colors-with-modes.css'
```

Comment:
Nit: Could we move stylesheets below JS imports and put annotations above them please?

Example: https://github.com/primer/brand/blob/7da349d21d7cc4bf77b105186a9a1aae14ae464b/packages/react/src/ActionMenu/ActionMenu.tsx#L27